### DISCUSSION TITLE
How can I turn off full screen mode on mobile or at all?

### DISCUSSION THREAD

**Discussion by zhen1asemen1uk** [cookpete/react-player#1794] [Q&A]
Be sure to [search for your issue](https://github.com/CookPete/react-player/issues) before opening a new one.

#### Current Behavior
On my iPhone 15 when I play the video, it's automatically open on fullscreen

#### Expected Behavior
When I play the video it just plying in defauld mode(NOT in FullScreen)

#### Steps to Reproduce

1. Click (play)
1. Full screen mode on(on my phone, on desktop - no)
1. Swipe down for close fullscreen mode( or click close fullscreen mode)
1. The video paused

#### Environment
- URL attempting to play: any
- Browser: safary, chrome, FB browser
- Operating system: iOS
- jsFiddle example: https://jsfiddle.net/sv5x3ug1

**Comment by samuelfuchs** [comment]
I just got here looking for the same solution

**Comment by nzhiti** [comment]
This can be done by settings playsinline to true if supported.

This is the documentation about that parameter on vimeo : https://help.vimeo.com/hc/en-us/articles/12425812053265-Inline-playback-on-mobile.

I don't know if it's supported by the video hoster you provided in the jsfiddle.

**Comment by shubham-khandelwal-sp** [comment]
On IOS, by default the popup will appear on the screen, as playsinline is by default set to false, please pass that as true to solve the issue.
